scraper.rb

```diff
@@ -0,0 +1,18 @@
+require 'net/http'
+require 'pry'
+require 'require_all'
+
+require_rel 'lib'
+
+url = URI.parse('http://www.senado.gov.ar/senadores/Historico/FechaResultado')
+
+params = {
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaDesde%5D%5Bday%5D' => 1,
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaDesde%5D%5Bmonth%5D' => 1,
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaDesde%5D%5Byear%5D' => 2015,
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaHasta%5D%5Bday%5D' => 1,
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaHasta%5D%5Bmonth%5D' => 1,
+  'senado_senadoresbundle_busquedahistoricostype%5BfechaHasta%5D%5Byear%5D' => 2016
+}
+
+page = MembersPage.new(response: Net::HTTP.post_form(url, params))
```

Comment:
I don't think this is going to work. It _appears_ to work at the moment because `Net::HTTP.post_form` _happens_ to return an object that responds to `#body`, but this parameter is supposed to be a `Scraped::Response` instance.

I think what you want to do is create a [custom request strategy class](https://github.com/everypolitician/scraped#custom-request-strategies) that wraps `Net::HTTP.post_form`'s return value. Something like this (untested!):

```ruby
class NetHttpPostRequest < Scraped::Request::Strategy
  def response
    Scraped::Response.new(url: url, body: Net::HTTP.post_form(url, config[:params]))
  end
end
```

Then you can call it like so:

```ruby
request = Scraped::Request.new(strategies: [{ strategy: NetHttpPostRequest, params: params }])
page = MembersPage.new(response: request.response)